### PR TITLE
Manage Link showing for organisers

### PR DESCRIPTION
**Pull request by sean-brydon**
## What does this PR do?

This PR allows the organisers/attendee[0] to be given the option to manage the event - reschedule/cancel 

Also hides info field if its an empty string. We were getting headings showing with no content (additional notes)

Fixes # (issue)

**Environment**: Staging(main branch) / Production

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: packages/emails/src/components/Info.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ export const Info = (props: {
   withSpacer?: boolean;
   lineThrough?: boolean;
 }) => {
-  if (!props.description) return null;
+  if (!props.description || props.description=="") return null;
```
Comment (sean-brydon) [inline packages/emails/src/components/Info.tsx:10]: Some cases like additionlNotes are set to "" and was causing a random header to appear in the email field

File: packages/emails/src/components/ManageLink.tsx [hunk 2]
```diff
@@ -4,27 +4,29 @@ import type { CalendarEvent, Person } from "@calcom/types/Calendar";
...
-        <>{t("need_to_reschedule_or_cancel")}</>
-      </p>
-      <p style={{ fontWeight: 400, lineHeight: "24px" }}>
-        <a href={getCancelLink(props.calEvent)} style={{ color: "#3e3e3e" }}>
-          <>{t("manage_this_event")}</>
-        </a>
-      </p>
-    </div>
-  );
+  if(props.attendee.email === props.calEvent.attendees[0].email || props.calEvent.organizer.email  === props.attendee.email ){
```
Comment (sean-brydon) [inline packages/emails/src/components/ManageLink.tsx:8]: I switch this if statement cause I couldn't for the life of me to get it to work and I'm not sure why :S

Comment (zomars) [reply packages/emails/src/components/ManageLink.tsx:8]: Let's figure out why and take the learning instead of just patching

**Review by pumfleet — APPROVED** [review]
Looks good to me

**Comment by zomars** [comment]
It wasn't ready to merge we found out some quirks on emails @baileypumfleet

**Pull request merged**